src/hooks/useSafeNotifications.ts

```diff
@@ -37,6 +42,44 @@ const useSafeNotifications = (): void => {
       dispatch(closeNotification({ id }))
     }
   }, [dispatch, chainId, safeAddress, implementationVersionState, version, chain?.shortName])
+
+  /**
+   * Show a notification when the Safe master copy is not supported
+   */
+
+  const masterCopy = safe.implementation.value
+
+  const [validMasterCopy] = useAsync(async () => {
```

Comment:
```suggestion
  const [validMasterCopy] = useAsync<boolean>(async () => {
```

Reply:
As `useAsync` uses generics, we need not strictly type this as we only call `getMasterCopies` which returns a `boolean`:

![image](https://user-images.githubusercontent.com/20442784/191962557-9ad55244-c5f1-40a5-8b51-cbc1ef156d8b.png)